core/workspace.js

```diff
@@ -490,8 +491,8 @@ Blockly.Workspace.prototype.fireChangeListener = function(event) {
  */
 Blockly.Workspace.prototype.getBlockById = function(id) {
   var block = this.blockDB_[id];
-  if (!block && this.getFlyout() && this.getFlyout().getWorkspace()) {
-    block = this.getFlyout().getWorkspace().blockDB_[id];
+  if (!block && this.getFlyout_() && this.getFlyout_().getWorkspace()) {
```

Comment:
So my solution was to replace all `getFlyout` -> `getFlyout_`, is that right?

If we do that, we should just make sure that we update `scratch-gui` (https://github.com/LLK/scratch-gui/blob/075216f585770f74a189ddaff66ac75a39c2308d/src/containers/blocks.jsx#L63-L66)